api/apiFunctions.ts

```diff
@@ -226,11 +227,6 @@ export async function getAllWeeklyPicks({
       return null;
     }
 
-    // check if any users have selected their pick
-    if (response.documents[0].userResults === '') {
```

Comment:
Why is this removed?

Reply:
It was duplicate. Please see rows 225 - 227.